system/BaseModel.php

```diff
@@ -1291,10 +1291,10 @@ protected function intToDate(int $value)
                 return $value;
 
             case 'datetime':
-                return date('Y-m-d H:i:s', $value);
+                return (string) Time::createFromTimestamp($value, app_timezone());
 
             case 'date':
-                return date('Y-m-d', $value);
+                return (string) Time::createFromTimestamp($value, app_timezone());
```

Comment:
Sorry, this is my mistake, but they are not the same. We must return as `'Y-m-d'`.